source/exploring_catalog_and_datasets/04_getting_involved/submitting_feedback.rst

```diff
@@ -0,0 +1,36 @@
+Submitting feedback
+===================
+
+.. admonition:: Important
+   :class: important
+
+   Since feedbacks are an optional feature, they are not activated on every OpenDataSoft portal. The activation of feedbacks entirely depends on the portal administrators.
```

Comment:
The activation of feedbacks entirely depends on the portal administrators.

DEPENDS ENTIRELY 😆